python/pyspark/pandas/namespace.py

```diff
@@ -407,6 +410,20 @@ def read_csv(
         index_spark_column_names = []
         index_names = []
 
+    data_spark_columns = [scol_for(sdf, col) for col in column_labels.values()]
+    if thousands is not None:
```

Comment:
I think you can set `locale` option in CSV. Should probably find one standard that adds `,` in numbers. e.g.) see https://docs.oracle.com/cd/E19455-01/806-0169/overview-9/index.html

Reply:
I think seems like pandas just simply replace the string specified by `thousands` parameter to empty string, if the column is possible to cast to numeric type, regardless of locale standard ??

For example,

```csv
name;age;job;money
Jorge;30;Developer;10000,,00,0,0,0,0
Bob;32;Developer;-1234,424,142424,0
```

```python
>>> pd.read_csv(path, sep=";", thousands=",")
    name  age        job           money
0  Jorge   30  Developer     10000000000
1    Bob   32  Developer -12344241424240
```